### DISCUSSION TITLE
Suggestion: Samsungs Devices with S-Pen

### DISCUSSION THREAD

**Discussion by krheinwald** [AppGround-io/bluetooth-keyboard-and-mouse-support#127] [Ideas]
Would love to be able to use my tablet with pen as a Windows pen for handwriting, drawing, etc. 

Should react to the pen only, without disturbance by accidental touches of the screen.

Extra features of the pen (touching the screen = left click, pen button = right click) would be great, so would be 'function soft keys* to e.g. bring up the Windows Whiteboard.

**Comment by AppGround-io** [comment]
This is definitely a feature I want to have as well, but I can not give a date yet when it will be implemented.

**Comment by parkerlreed** [comment]
Loving the recently added pen support. 

Not sure if possible over just Bluetooth HID, but would be great to have it as an absolute drawing tablet. Maybe an option in the pen settings to set the target device screen resolution and then map the touchpad area 1:1 when using the pen.

**Reply by QuiXilver777** [reply]
I sent some feedback about this already, but I'd also really like an option to use the S-Pen's gyroscope and accelerometer as the airmouse. The app Smouse shows the proof of concept, and the S-Pen SDK seems simple enough to use. I don't think it will be that hard to implement and it probably won't be a niche feature either, I know that I'd have bought premium entirely just for this feature. It's extremely useful for Dex. Just a cool idea I'd really like to see.
